cloud-tasks/function/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "email-function",
+  "description": "Cloud Function sends an email when triggered by Cloud Task",
+  "main": "index.js",
+  "license": "Apache-2.0",
```

Comment:
nit: we don't really need a license or author as this is a private module and we already declare the license in the parent package.

Reply:
done.